src/__tests__/config/pools.test.ts

```diff
@@ -29,8 +29,15 @@ describe('Config pools', () => {
   it.each(poolsToTest.filter((pool) => pool.stakingToken.symbol !== 'BNB'))(
     'Pool %p has the correct staking token',
     async (pool) => {
-      const contract = getSouschefContract(pool.sousId)
-      const stakingTokenAddress = await contract.methods.syrup().call()
+      let stakingTokenAddress = null
+      try {
+        const contract = getSouschefV2Contract(pool.sousId)
+        stakingTokenAddress = await contract.methods.stakedToken().call()
+      } catch (error) {
```

Comment:
Is the `contract.methods.stakedToken()` call compatible (in theory) with _all_ pools, or just the new ones? 

Just wondering whether it's going to attempt this call, error, and then attempt the `syrup` call for every pool other than the new ones?

Reply:
syrup has been replaced by stakedToken in the new contract, it's basically the only breaking change on the abi, and it was used only in the tests. I didn't find a cleaner solution , but if you have any suggestion you can propose a different implementation